validate/validate.go

```diff
@@ -301,12 +301,23 @@ func validateTdAttributes(value []byte, fixed1, fixed0 uint64) error {
 	return nil
 }
 
-// TdxQuote validates fields of the protobuf representation of an attestation Quote against
-// expectations. Does not check the attestation certificates or signature.
```

Comment:
> Does not check the attestation certificates or signature
As this is the public API, the above line should not be removed even if it is repeated for the internal function below.